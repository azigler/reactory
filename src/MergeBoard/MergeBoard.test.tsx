import React from "react"
import { createRoot } from "react-dom/client"
import { act } from "react-dom/test-utils"
import { MergeBoard } from ".."

describe("MergeBoard component", () => {
  it("renders", () => {
    const container = document.createElement("main")
    const root = createRoot(container)
    act(() => {
      root.render(<MergeBoard />)
    })
    expect(container).toMatchSnapshot()
  })
})
