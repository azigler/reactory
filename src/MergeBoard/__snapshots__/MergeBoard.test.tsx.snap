// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MergeBoard component renders 1`] = `
<main>
  <div>
    MergeBoard
  </div>
</main>
`;
